### PR TITLE
Graphql subscription instrumentation

### PR DESCRIPTION
**Pull request by willianpc**
This PR adds the `subscription` instrumentation to the GraphQL instrumentation

### CODE REVIEW

**Review by sanojsubran — CHANGES_REQUESTED** [review]
Great work @willianpc !! I have mentioned some minor things as review comments. Please check it.

File: instrumentation/instagraphql/README.md [hunk 1]
```diff
@@ -17,10 +19,15 @@ $ go get github.com/instana/go-sensor/instrumentation/instagraphql
 Usage
 -----
 
-`instagraphql` offers a function wrapper for the [`graphql.Do()`][instagraphql.Do] method capable of collecting the relevant
-data about the GraphQL query and sending them to the Instana backend.
+A complete working example can be found [here](example).
+
+`instagraphql` offers the following method wrappers able to collect GraphQL query data:
```
[the remark this answers is not in the document]
Comment (willianpc) [reply instrumentation/instagraphql/README.md:24]: This is a pattern that is copied over and over from other instrumentations.
Let me keep it this way for now, but I agree with you that we should review this.
Maybe we make an effort to rewrite this for all instrumentations that follow this confusing pattern.

Is that ok for you?

[the remark this answers is not in the document]
Comment (sanojsubran) [reply instrumentation/instagraphql/README.md:24]: Sounds good !!

**Review by sanojsubran — APPROVED** [review]
Great work !!

**Pull request merged**